packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss

```diff
@@ -51,15 +51,21 @@
 
 .headerContent {
   @include type.type-style('heading-compact-02');
+  align-items: center;
+  border: 0.125rem solid transparent;
   color: $ui-05;
+  outline: none;
+  padding: 0 2rem 0 1rem;
+  text-decoration: none;
+  user-select: none;
+  vertical-align: center;
 }
 
-.closeButton {
-  background-color: $ui-02;
-  margin-right: spacing.$spacing-05;
-
+.closeButton,
+.closeButton:hover {
+  background-color: #fff;
```

Comment:
Should this just be `white`? It's more consistent with the SVG fill.

Reply:
Just copying this from somewhere else, but that makes sense, will switch.